app/components/RegisterCard/index.js

```diff
@@ -1,34 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { client } from '../../graphql/apolloClient';
+
+async function doMutation(registerBody) {
+	const { data } = await client.mutate({
+		mutation: registerUser,
+		variables: {
+			registerBody: registerBody,
+		},
+	});
+	return data
+}
 
 export const RegisterCard = () => {
+	const [username, setUsername] = useState();
+	const [usertype, setUsertype] = useState();
+	const [password, setPassword] = useState();
+	const [fullname, setFullname] = useState();
+	const [idDoc, setIdDoc] = useState();
+	const responseBody = {
+		username: '',
+		usertype: '',
+		password: '',
+		fullname: '',
+		idDoc: '',
```

Comment:
```suggestion
		document: 0,
```
Estos nombres deben ser consistentes con los del graphQLSchema